app/cothorityd/cothorityd.go

```diff
@@ -96,3 +108,10 @@ func runServer(ctx *cli.Context) {
 
 	server.RunServer(config)
 }
+
+// checkConfig contacts all servers and verifies if it receives a valid
+// signature from each.
+func checkConfig(c *cli.Context) error {
```

Comment:
Yes, I thought of merging it directly into the `lib/server`, but then the "-g"-flag would be hardcoded, so I didn't.